app/src/main/java/com/github/libretube/MainActivity.kt

```diff
@@ -40,6 +42,15 @@ class MainActivity : AppCompatActivity() {
         RetrofitInstance.url = sharedPreferences.getString("instance", "https://pipedapi.kavin.rocks/")!!
         DynamicColors.applyToActivitiesIfAvailable(application)
         setContentView(R.layout.activity_main)
+        var languageName = sharedPreferences.getString("language", "en")!!
+        if (languageName != "") {
+            val locale = Locale("$languageName")
+            val res = resources
+            val dm = res.displayMetrics
+            val conf = res.configuration
+            conf.locale = locale
+            res.updateConfiguration(conf, dm)
+        }
```

Comment:
I couldn't push to your branch, but these were the changes I wanted to make:
![image](https://user-images.githubusercontent.com/20838718/167009117-5deb5ca0-c5ac-4687-a6d8-aa67047c8098.png)

This will use the language for timeago too.